brambling/tests/functional/test_finance_table.py

```diff
@@ -0,0 +1,57 @@
+from django.test import TestCase
+
+from brambling.utils.model_tables import Cell
+from brambling.views.utils import FinanceTable
+from brambling.tests.factories import TransactionFactory, EventFactory, \
+    PersonFactory, OrderFactory
+
+
+class FinanceTableTestCase(TestCase):
+
+    def setUp(self):
+        self.order = OrderFactory(code='TK421')
+        self.event = EventFactory()
+        self.transactions = [TransactionFactory(order=self.order)]
+
+    def test_assert_true(self):
+        assert True
```

Comment:
Hah, this was my test to make sure the test runner had found my file (which initially it didn't)! Seems worth deleting.